packages/utils/src/cache/helpers.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+
+import { MESSENGER_LAST_MESSAGE_INTERVAL } from './constants.js';
+
+type LoggerTime = {
+  startAt: number
+  endAt?: number
+}
+export class Logger {
```

Comment:
This file is just a logger. I think it should be rename as `Logger.ts` or `helper/Logger.ts`.